lib/ansible/modules/web_infrastructure/jenkins_job_facts.py

```diff
@@ -170,9 +172,8 @@ def get_jenkins_connection(module):
 
 def test_dependencies(module):
     if not HAS_JENKINS:
-        module.fail_json(
-            msg="python-jenkins required for this module. "
-            "see http://python-jenkins.readthedocs.io/en/latest/install.html")
```

Comment:
URL here too.

Reply:
Thanks, I added in the url arg near the end and forgot to go back and re-add the ones I removed.